### ISSUE TITLE
RRFS filename changes needed

### ISSUE THREAD

**Issue by MatthewPyle-NOAA** [NOAA-EMC/rrfs-workflow#281]
As part of preparing for the EE2/kickoff meeting with NCO, have documented some of the filename changes that will be needed.

conus_3km to conus:

`rrfs.tCCz.prslev.fFFF.conus_3km.grib2` ==> `rrfs.tCCz.prslev.fFFF.conus.grib2`

Ensemble members need to include member number (https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf, page 6):

`rrfs.tCCz.prslev.fFFF.grib2` ==> `rrfs.tCCz.mMM.prslev.fFFF.grib2`

And a directory reorganization is also being considered, to eliminate the refs.${PDY} structure and place everything under rrfs.${PDY}

**Comment by BenjaminBlake-NOAA** [comment]
@MatthewPyle-NOAA  I can do this one next, they seem like fairly straightforward changes that shouldn't be too involved.

**Comment by MatthewPyle-NOAA** [comment]
Thanks @BenjaminBlake-NOAA   Only big item will be notifying downstream users about it before making the switch in the real-time system.

**Comment by BenjaminBlake-NOAA** [comment]
@MatthewPyle-NOAA  Agreed, we'll have to communicate that when we're ready to update the real-time parallel.  I have these changes ready and was able to test the "conus_3km" to "conus" change, but haven't tested the changes for adding the ensemble member numbers.  Maybe that would be easiest to test as a local change in the real-time system?  I'll open a PR on Monday and we can discuss further.
